frontend/public/index.html

```diff
@@ -18,7 +18,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="The Programming Club of IIITDM Jabalpur"
```

Comment:
The name of the club is already present in the title of the link preview:

<img width="283" alt="image" src="https://github.com/BitByte-TPC/TPC-website/assets/39924567/d9c12a4c-211c-4b2c-a4c1-b03ea0ec6c7b">

The description meta tag should instead contain a short description of the website, or maybe just "Official website of The Programming Club, IIITDM Jabalpur"?